lam4-cli/app/Main.hs

```diff
@@ -113,6 +114,10 @@ main = do
       putStr $ T.unpack $ ToSimala.render simalaProgram
       print "-------------------------------"
 
+      putStrLn "-------- Natural language (sort of) ---------"
```

Comment:
This is OK for now. But as mentioned on the call, I will try to expand the CLI so that one can tell it whether to even bother generating the natural language; and if so, where to export the generated natural language to